gui/sharing/models.py

```diff
@@ -43,6 +43,13 @@ class CIFS_Share(Model):
         verbose_name=_('Use as home share'),
         default=False,
     )
+    cifs_timemachine = models.BooleanField(
+        verbose_name=_('Time Machine'),
+        help_text=_(
+            'Enable Time Machine backups on this share.'
+        ),
+        default=False,
+    )
```

Comment:
If time machine is a global option (needs to be enabled for all shares), shouldnt this be in service settings instead?

Reply:
I think I misunderstood, only vfs_fruit needs to be enabled on all. time machine knob is only in the share in question, right?

Reply:
That's correct. The knob has the same function as the AFP one.